spec/paths/reports@dashboard.yaml

```diff
@@ -0,0 +1,46 @@
+get:
+  tags:
+    - Reports
+  summary: Retrieve dashboard data
+  description: Retrieve dashboard data
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start day
+      required: true
+      type: string
+      format: date-time
+    - name: periodEnd
+      in: query
+      description: Report end day
+      required: true
+      type: string
+      format: date-time
+    - name: metrics
+      in: query
+      description: Comma-separated list of metrics
+      required: true
+      type: string
+      format: date-time
+    - name: segments
+      in: query
+      description: Dashboard report segments as a JSON array
+      required: true
```

Comment:
Again, I don't think this should be required.  If I recall this is a complex object to create.  I think it should default to "all".